### PR TITLE
Add public A and PTR records for `vulnscan` instances

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This pull request adds public A and PTR records for the `vulnscan` instances.

## 💭 Motivation and context ##

We need live A and PTR records to help our `vulnscan` instances look legitimate. This is part of an effort to update our Nessus SMTP configuration for scanning to avoid our `vulnscan` instances getting put on spamlists (such as [Spamhaus](https://www.spamhaus.org/)).

## 🧪 Testing ##

Automated tests pass. I also verified that deploying and redeploying a `vulnscan` instances in my testing environment successfully created and updated both public records.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: terraform/locals.tf [hunk 1]
```diff
@@ -86,6 +86,18 @@ locals {
...
+  # Since our elastic IPs are handled differently in
+  # production vs. non-production workspaces, their corresponding
+  # Terraform resources (data.aws_eip.cyhy_nessus_eips,
+  # data.aws_eip.cyhy_nessus_random_eips) may or may not be created.  To
+  # handle that, we use coalescelist() to choose the (non-empty) list
+  # containing the valid eip objects.
+  nessus_public_ips = coalescelist(
+    data.aws_eip.cyhy_nessus_eips,
+    aws_eip.cyhy_nessus_random_eips,
+  )
```
Comment (dav3r) [inline terraform/locals.tf:99]: As long as we're making this a local variable, we should utilize it here too:
https://github.com/cisagov/cyhy_amis/blob/2d2a70bc3bc51cacce4e0babd17cec3bb5363d6f/terraform/cyhy_nessus_ec2.tf#L106-L109

**Review by dav3r — APPROVED** [review]
👍 👍

**Review by jsf9k — APPROVED** [review]
This all looks good to me, although I agree with @dav3r's one comment.

File: terraform/locals.tf (hunk 1 above)
Comment (mcdonnnj) [reply terraform/locals.tf:99]: Please see [80d489a](https://github.com/cisagov/cyhy_amis/pull/425/commits/80d489a197e72d874000db8860f4581c65845767).

Comment (jsf9k) [reply terraform/locals.tf:99]: I'm going to speak for @dav3r here (since he's out today) and say that commit 80d489a fully addresses his comment.

Comment (dav3r) [reply terraform/locals.tf:99]: Thanks @jsf9k for covering for me here.  You were indeed correct - this commit looks great to me.

**Pull request merged**